packages/btfs-http-client/src/signbatch/index.js

```diff
@@ -0,0 +1,71 @@
+'use strict'
+
+const configure = require('../lib/configure')
+const toCamel = require('../lib/object-to-camel')
+const protoGuard = require('../../protos/guard_pb')
+const protoEscrow = require('../../protos/escrow_pb')
+const peerId = require('peer-id')
+
+var signContract = function (privKey, contract, sessionStatus) {
+  return new Promise(   async (resolve, reject) => {
+      var by = Buffer.from(contract,'base64') // contract in string <- 1. OK
+      const id = await peerId.createFromPrivKey(Buffer.from(privKey, 'base64'))
+      let raw
+      if (sessionStatus == 'initSignReadyEscrow' || sessionStatus == "initSignProcessEscrow"){
+          raw = proto.escrow.EscrowContract.deserializeBinary(by).serializeBinary()
+      }
+      else if (sessionStatus == 'initSignReadyGuard' || sessionStatus == "initSignProcessGuard") {
+          raw = proto.guard.ContractMeta.deserializeBinary(by).serializeBinary()
+      }
+     const signature = await id.privKey.sign(Buffer.from(raw,'base64'))
+     var signC = (signature.toString('base64'))
+     resolve(signC);
```

Comment:
be consistent with `;`, either use them or don't. JavaScript adds `;` automatically so its not necessary to use them, but stick with one style to be consistent. Our team uses `;` but its up to you guys.

Reply:
okay, will do.